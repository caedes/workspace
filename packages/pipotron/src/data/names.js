export default [
  'la restriction',
  "l'orientation",
  'la crise',
  "l'inertie",
  'la difficulté',
  "l'austérité",
  'la dégradation',
  'cette rigueur',
  'la dualité de la situation',
  'la baisse de confiance',
  'la morosité',
  'la situation',
  "l'ambiance",
  'la politique',
  'la fragilité',
  'complexité',
  "l'inconstance",
  'cette inflexion',
];
