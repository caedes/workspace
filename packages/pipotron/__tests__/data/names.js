import names from '../../src/data/names';

describe('names', () => {
  it('returns the names', () => {
    expect(names).toMatchSnapshot();
  });
});
