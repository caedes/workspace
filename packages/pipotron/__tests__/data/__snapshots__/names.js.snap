// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`names returns the names 1`] = `
Array [
  "la restriction",
  "l'orientation",
  "la crise",
  "l'inertie",
  "la difficulté",
  "l'austérité",
  "la dégradation",
  "cette rigueur",
  "la dualité de la situation",
  "la baisse de confiance",
  "la morosité",
  "la situation",
  "l'ambiance",
  "la politique",
  "la fragilité",
  "complexité",
  "l'inconstance",
  "cette inflexion",
]
`;
